liquibase-core/src/test/groovy/liquibase/configuration/core/EnvironmentValueProviderTest.groovy

```diff
@@ -1,87 +1,14 @@
 package liquibase.configuration.core
 
 import liquibase.GlobalConfiguration
-import liquibase.Scope
 import liquibase.hub.HubConfiguration
-import liquibase.license.LicenseInstallResult
-import liquibase.license.LicenseService
-import liquibase.license.LicenseServiceFactory
-import liquibase.license.Location
 import spock.lang.Specification
 import spock.lang.Unroll
 
 class EnvironmentValueProviderTest extends Specification {
-    static boolean licenseValid = false;
-    static final mockLicenseService = new LicenseService() {
```

Comment:
This is removing unnecessary tests which fail on environments without a "java.home" environment variable (most people).